resources/migrations/001_update_migrations.yaml

```diff
@@ -8719,7 +8719,7 @@ databaseChangeLog:
         - not:
             - foreignKeyConstraintExists:
                 foreignKeyName: fk_pulse_channel_channel_id
-                foreignKyeTableName: pulse_channel
+                foreignKeyTableName: pulse_channel
```

Comment:
woah. this is kinda nuts

Reply:
yeah...I'm not sure what the existing behavior is with that typo

Reply:
actually, these typos were just in preconditions, so it shouldn't have affected the actual migration logic. but I presume the preconditions wouldn't have been enforced properly

Reply:
Wow, yea this could be some evidence that we should improve linting on this file.

Reply:
Found another here: https://github.com/metabase/metabase/blob/master/resources/migrations/001_update_migrations.yaml#L9760

~Not sure if we should actually fix it here.~ @noahmoss fixed it in this pr 😎